src/main/java/io/kestra/plugin/serdes/json/IonToJson.java

```diff
@@ -103,44 +110,52 @@ public Output run(RunContext runContext) throws Exception {
         File tempFile = runContext.workingDir().createTempFile(suffix).toFile();
         URI from = new URI(runContext.render(this.from).as(String.class).orElseThrow());
 
-        var renderedCharset = runContext.render(this.charset).as(String.class).orElseThrow();
+        var renderedCharsetName = runContext.render(this.charset).as(String.class).orElseThrow();
+        Charset outputCharset = Charset.forName(renderedCharsetName); // Get the Charset object
+
+        // Create a standard Jackson ObjectMapper for JSON output
+        ObjectMapper jsonObjectMapper = new ObjectMapper()
+            .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+            .setSerializationInclusion(JsonInclude.Include.ALWAYS)
```

Comment:
when you're at it, creating object mapper is costly and they are safe to use by multiple threads concurrently.
What we usually do is to init it as a constant, and the, if needed customization `copy()` it and do the customization on the copy.
Not sure it could be done it but worth looking at it.